client/src/components/Home.js

```diff
@@ -1,127 +1,83 @@
-                return (
-                  <div>
-                    <Row
-                      className="testimonial-block"
-                      type="flex"
-                      justify="center"
-                      align="middle"
-                    >
-                      <Col
-                        className="testimonial-block__left"
-                        span={6}
-                        offset={1}
-                      >
-                        <img
-                          className="testimonial-block__left__img"
-                          src={element.picture}
-                          alt=""
-                        />
-                      </Col>
-                      <Col
-                        className="testimonial-block__right"
-                        span={12}
-                        offset={1}
-                      >
-                        <h1>{element.person}</h1>
-                        <h3>{element.title}</h3>
-                        <p>{element.testimonial}</p>
-                      </Col>
-                    </Row>
-                  </div>
-                );
-              })}
-            </Carousel>
-          </Col>
-        </Row>
-        <Row
-          className="home-block-4"
-          type="flex"
-          justify="center"
-          align="middle"
-        >
-          <Col>
-            <Row type="flex" justify="center" align="middle">
-              <Col>
-                <h1>Partners in the Community</h1>
-              </Col>
-            </Row>
-            <Row type="flex" justify="center" align="middle">
-              <Col className="home-block-4__partner" span={4}>
-                <img
-                  src="/asset/partners/cu_fair.jpg"
-                  alt="Champaign-Urbana Fair"
-                />
-              </Col>
-              <Col className="home-block-4__partner" span={4}>
-                <img
-                  src="/asset/partners/cuphd.png"
-                  alt="Champaign-Urbana Public Health District"
-                />
-              </Col>
-              <Col className="home-block-4__partner" span={4}>
-                <img src="/asset/partners/dhai_tree.jpg" alt="Dhairtree" />
-              </Col>
-              <Col className="home-block-4__partner" span={4}>
-                <img
-                  src="/asset/partners/three_spinners.jpg"
-                  alt="Three Spinners"
-                />
-              </Col>
-              <Col className="home-block-4__partner" span={4}>
-                <img src="/asset/partners/trc.jpg" alt="The Refugee Center" />
-              </Col>
-            </Row>
-          </Col>
-        </Row>
-      </>
-    );
-  }
-}
+import { HomeBlock2Desktop, HomeBlock3Desktop } from './desktop/HomeDesktop';
+import { HomeBlock2Mobile, HomeBlock3Mobile } from './mobile/HomeMobile';
+
+const Home = () => {
+  const [{ height, width }, isMobile] = useWindowDimensions();
+
+  return (
+    <>
```

Comment:
This fragment shouldn't be necessary _(I think ??)_. Try removing this wrapper and if it breaks anything leave it in, otherwise its better for indentation :)

Reply:
wait i don't understand what this means can u explain

Reply:
Basically there's the extra wrapper (`<>` *code* `</>`) around the `Row` component. Normally they're used when there needs to be multiple components returned (such as `<> <component 1 /> <component 2 />  </>`), but since there's only one here we probably don't need it

Reply:
oh wait we do need this since there are multiple parallel components underneath it

Reply:
yes didnt see that :)